Simplified/Reader2/ReaderStackConfiguration/LibraryService.swift

```diff
@@ -15,6 +15,43 @@ import UIKit
 import R2Shared
 import R2Streamer
 
+// =============================================================================
+// MARK: - Workaround for VoiceOver Issues
+
+/// Temporary workaround to continue using the old Readium `PublicationServer`.
+///
+/// Readium 2.5.0 deprecates `PublicationServer` and introduces
+/// `GCDHTTPServer` in its place. However, this causes significant scrolling
+/// and book content navigation issues with VoiceOver. By casting your
+/// `PublicationServer` references to `PublicationServing`, this workaround
+/// silences the deprecation warnings until we find an official solution.
+protocol PublicationServing: ResourcesServer {
+  func add(_ publication: Publication) throws
+  func removeAll()
+}
+
+extension PublicationServer: PublicationServing {
```

Comment:
this still emits a warning because `PublicationServer` as a class is deprecated. I don't see a way to work around this, but if you do let me know.